### DISCUSSION TITLE
Making changes to note in Anki get deleted.

### DISCUSSION THREAD

**Discussion by whatevkev** [ObsidianToAnki/Obsidian_to_Anki#605] [Q&A]
Basically edits I make in Anki are being erased when the plugin re-scans in obsidian.  Looking for work around for this.  In obsidian after the ID code pops up and card is sent to ANKI can i just delete the paragraph to prevent the plugin from erasing any changes I make in ANKI without deleting the card itself? (By delete I don't mean putting DELETE at the end of the paragraph to have the note removed from anki).      

How I am using plugin: As I take notes I make Anki cards and get them sent over to my Anki.  I pretty much exclusively use cloze deletions and use the curlycloze inline.  This is  applied: ((?:.+\n)*(?:.*{.*)(?:\n(?:^.{1,3}$|^.{4}(?<!<!--).*))*).  The main reason I use this plugin is because the ease of making Anki notes as I take regular notes, and how each card can now be linked to obsidian note.  

The problem is that when I review anki cards I tend to make several edits to the cards sent over from obsidian.  Specifically changing the formatting, highlighting, underlining things in Anki.  I also add tags, fill in fields (my ANKI cards have about 10 fields, using the AnKing cards).  When I go back in obsidian and make more cards, click the sync button it adds the new cards but erases all the changes I made in Anki on older cards. 

Thanks

**Comment by envico801** [comment]
Hi @whatevkev, I understand what you mean, I don't know if there is a way to send the changes you make in Anki to Obsidian [Anki ➡️ Obsidian]. It would be better to make the changes in Obsidian and sync them there, so they are saved in Anki too [Obsidian ➡️ Anki].

Maybe you can do this using some [Add-on](https://ankiweb.net/shared/addons) of Anki itself, [Obsidian_to_Anki](https://github.com/ObsidianToAnki/Obsidian_to_Anki) uses [AnkiConnect](https://ankiweb.net/shared/info/2055492159) to make the changes (it uses an API to simplify the actions), maybe you could create an addon on the Anki side that synchronizes the changes with a click of a button for example.

**Edit**: I forgot to add that every time you sync in obsidian, it will update the content of the cards, and it is for this reason that the changes you make in Anki itself are deleted.

To prevent it from updating the cards you have in Anki you can delete the ID of the card so it can't find it and prevent it from deleting your changes in Anki. The problem with doing this is that the changes you make in obsidian will be useless or you will have to re-create the cards from scratch.
